packages/api/endpoints/logs.js

```diff
@@ -20,10 +39,18 @@ function convertLogLevelForQuery(query) {
  * @returns {Promise<Object>} the promise of express response object
  */
 async function list(req, res) {
```

Comment:
We should probably unit test list/get to cover the new logic  in such a way that the right configuration is sent to search and mock out the return (rewire?), in lieu of trying to run a test like L 100-110 and populating a local elastic search somehow (yuck).     Really that's wrapping/testing the logic in `const esConfig = () => (process.env.log_destination_arn ? metricsConfig() : cumulusConfig());` so it seems trivial, but that's the logic change here.

Reply:
Tests added.

Reply:
Added tests.